release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -966,6 +966,24 @@ While storage class referencing to the in-tree storage plugin will continue work
 
 For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
 
+[id="ocp-4-12-storage-vsphere-auto-migration"]
+==== Upgrading from {product-title} 4.12 to 4.13 and later with vSphere in-tree PVs
+If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from {product-title} 4.12 to 4.13 or later, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} upgrade is blocked. After updating vSphere, your {product-title} upgrade can occur and automatic CSI migration for vSphere only occurs:
+
+* For 4.13: only if you opt in.
+
+* For 4.14 and later: automatically by default.
+
+Alternatively, if you do not want to update vSphere, you can proceed with an {product-title} upgrade by either:
+
+* Initiating the upgrade from a {product-title} 4.12 Z-stream.
+
+* Performing an administrator acknowledgment
+
+If you upgrade to {product-title} 4.13 or later without updating vSphere, and then, for 4.13, opt in to migration, known issues can occur. link:https://access.redhat.com/node/7011683[Before upgrading, carefully read this knowledge base article].
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc#persistent-storage-csi-migration-vsphere_persistent-storage-csi-migration[Upgrading from {product-title}4.12 to 4.13 and later].
```

Comment:
The link doesn't add much, only the admin ack command. Is it worth adding the command here, or reducing the release note to a base minimum? 

> If you are using vSphere in-tree persistent volumes (PVs) and want to upgrade from OpenShift Container Platform 4.12 to 4.13 or later, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the OpenShift Container Platform upgrade is blocked. After updating vSphere, your OpenShift Container Platform upgrade and automatic CSI migration can occur. For more information, see [Upgrading from OpenShift Container Platform4.12 to 4.13 and later](https://65546--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-migration#persistent-storage-csi-migration-vsphere_persistent-storage-csi-migration).

Reply:
If you want to leave them both, maybe a snippet, in case something changes?